apps/mn-synt-arbeidsforhold-service/src/main/java/no/nav/registre/testnorge/mn/syntarbeidsforholdservice/consumer/command/GenerateNextArbeidsforholdCommand.java

```diff
@@ -0,0 +1,33 @@
+package no.nav.registre.testnorge.mn.syntarbeidsforholdservice.consumer.command;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+import java.util.concurrent.Callable;
+
+import no.nav.registre.testnorge.libs.dependencyanalysis.DependencyOn;
+import no.nav.registre.testnorge.libs.dto.syntrest.v1.ArbeidsforholdRequest;
+import no.nav.registre.testnorge.libs.dto.syntrest.v1.ArbeidsforholdResponse;
+
+@DependencyOn("syntrest")
+@RequiredArgsConstructor
+public class GenerateNextArbeidsforholdCommand implements Callable<ArbeidsforholdResponse> {
+    private final WebClient webClient;
+    private final ArbeidsforholdRequest arbeidsforholdDTO;
+
+    @Override
+    public ArbeidsforholdResponse call() {
+        return webClient
+                .post()
+                .uri("/api/v1/generate/arbeidsforhold/sklearn")
```

Comment:
For syntrest er uri: "/api/v1/generate/amelding/arbeidsforhold/sklearn".  (eventuelt "/api/v1/generate/amelding/arbeidsforhold" hvis du heller vil ta i bruk neural network modellen)